source/_integrations/starlink.markdown

```diff
@@ -59,3 +61,7 @@ The Starlink integration allows you to integrate your [Starlink](https://www.sta
 ### Switch
 
 - Stowed - Controls whether Dishy is stowed
+
+### Device Tracker
+
+- Device location - Tracks the location of Dishy. Note you need to allow location access on the local network via the Starlink app for this to work.
```

Comment:
Should probably mention that the entity is disabled by default and needs to be enabled once it's been enabled via the app.

Reply:
I mentioned it is disabled by default in both the Starlink app and HomeAssistant. Do you think that's sufficient?